crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -556,6 +557,17 @@ module sui::sui_system {
         self.safe_mode = true;
     }
 
+    public entry fun checkpoint_prologue(
+        clock: &mut Clock,
```

Comment:
Do we turn off the entry rule for `sui` functions? I forget

Reply:
I make that change in this PR